addons/linecompass/script_component.hpp

```diff
@@ -0,0 +1,37 @@
+#define COMPONENT linecompass
+#define COMPONENT_BEAUTIFIED Line Compass
+#include "\z\diwako_dui\addons\main\script_mod.hpp"
+
+// #define DEBUG_MODE_FULL
+#define DISABLE_COMPILE_CACHE
+
+#ifdef DEBUG_ENABLED_INDICATORS
+    #define DEBUG_MODE_FULL
+#endif
+    #ifdef DEBUG_SETTINGS_INDICATORS
```

Comment:
```suggestion
#ifdef DEBUG_SETTINGS_INDICATORS
```

Reply:
will change once all the other things are done